service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ConfirmApplicationReviewedControllerMidEventTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.AdditionalApplicationsBundle;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.event.ConfirmApplicationReviewedEventData;
+import uk.gov.hmcts.reform.fpl.service.additionalapplications.ConfirmApplicationReviewedService;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@WebMvcTest(ConfirmApplicationReviewedController.class)
+@OverrideAutoConfiguration(enabled = true)
+public class ConfirmApplicationReviewedControllerMidEventTest extends AbstractCallbackTest {
+    @MockBean
+    private ConfirmApplicationReviewedService confirmApplicationReviewedService;
+
+    private static final Element<AdditionalApplicationsBundle> APPLICATION_BUNDLE_ELEMENT =
+        element(AdditionalApplicationsBundle.builder()
+            .uploadedDateTime("2022 Dec 30")
```

Comment:
just wondering if the date time format should be like "1 January 2021, 12:00pm"? 

I took this as an example `MessageJudgeControllerAboutToStartTest.shouldInitialiseCaseFieldsWhenAdditionalApplicationsDocumentsAndJudicialMessagesExist`

Reply:
updated